unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenTipsView.cs

```diff
@@ -0,0 +1,54 @@
+using Cysharp.Threading.Tasks;
+using System;
+using System.Threading;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+namespace DCL.LoadingScreen
+{
+    /// <summary>
+    /// View responsible of showing the corresponding provided tip by LoadingScreenTipsController
+    /// </summary>
+    public class LoadingScreenTipsView : MonoBehaviour
+    {
+        private readonly TimeSpan SHOWING_TIME_TIPS = TimeSpan.FromSeconds(2);
+
+        [SerializeField] private TMP_Text tipsText;
+        [SerializeField] private Image tipsImage;
+
+        private LoadingScreenTipsController tipsController;
+        private CancellationTokenSource disposeCts = new ();
+
+        public void SetLoadingTipsController(LoadingScreenTipsController tipsController)
```

Comment:
I wonder if we can reverse control so `Controller` will control `View` and not vice versa. it will look more natural and aligned with "correct" controllers.
Moreover, you already create `TipsController` from the `LoadingScreenController` and invoke fading in/our logic from there so you can control `TipsController` from there.

Reply:
Reversed! Definitely cleaner that way.